internal/elasticsearch/cluster/settings_test.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 func TestAccResourceClusterSettings(t *testing.T) {
-	resource.UnitTest(t, resource.TestCase{
```

Comment:
> // UnitTest is a helper to force the acceptance testing harness to run in the
// normal unit test suite. This should only be used for resource that don't
// have any external dependencies.
https://github.com/hashicorp/terraform-plugin-sdk/blob/44ccfdce81e30da9672c731270773a2510772796/helper/resource/testing.go#L755-L765

`UnitTest` is inappropriate because the resources depend on elasticsearch.